lmvachon/bad.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<title>Lauren's Bad Code Example</title>
+
+	<link rel="stylesheet" href="/css/style.css" type="text/css" media="all" />
+</head>
+
+<body>
+
+	<header>
+		<nav>
+			<a href="#">Home
+			<a href="#">About
+			<a href="#">Contact
+		</nav>
+	</header>
+	<section>
```

Comment:
I'd consider eliminating this section tag. I could be wrong, but my understanding of HTML5 is that `<section >` and `<article >` are usually interchangeable, one substituting for the other rather than nesting inside the other. On section: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section ("If it makes sense to separately syndicate the content of a <section> element, use an <article> element instead.") In the end, though, I don't see this as a big deal. :)